docs/source/guides/logical_types_and_semantic_tags.ipynb

```diff
@@ -0,0 +1,693 @@
+    "    'ages_nullable' : [None, 2, 22, 33]\n",
+    "})\n",
+    "\n",
+    "numerics_df.ww.init(logical_types={'ages':'Age', 'ages_nullable':'AgeNullable'})\n",
+    "numerics_df.ww"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "forty-banks",
+   "metadata": {},
+   "source": [
+    "#### Categorical Logical Types\n",
+    "\n",
+    "##### Categorical \n",
+    "\n",
+    "Represents a Logical Type with few unique values relative to the size of the data.\n",
+    "\n",
+    "- **physical type**: `category`\n",
+    "- **inference**: Woodwork defines a threshold for percentage unique values relative to the size of the series below which a series will be considered categorical. See [setting config options guide](setting_config_options.ipynb#Categorical-Threshold) for more information on how to control this threshold.\n",
+    "- **koalas note**: Koalas does not support the `category` dtype, so for Koalas DataFrames and Series, the `string` dtype will be used.\n",
+    "\n",
+    "\n",
+    "Some examples of data for which the Categorical logical type would apply:\n",
+    "\n",
+    "- Gender\n",
+    "- Eye Color\n",
+    "- Nationality\n",
+    "- Hair Color\n",
+    "- Spoken Language\n",
+    "\n",
+    "##### CountryCode\n",
+    "\n",
+    "Represents Logical Types that use the ISO-3166 standard country code to represent countries. ISO 3166-1 (countries) are supported. These codes should be in the Alpha-2 format.\n",
+    "\n",
+    "- **physical type**: `category`\n",
+    "- **standard tags**: `{'category'}`\n",
+    "- **koalas note**: Koalas does not support the `category` dtype, so for Koalas DataFrames and Series, the `string` dtype will be used.\n",
+    "\n",
+    "For example: `'AU'` for Australia, `'CN'` for China, and `'CA'` for Canada.\n",
+    "\n",
+    "##### Ordinal\n",
+    "\n",
+    "A Ordinal variable type can take ordered discrete values. Similar to Categorical, it is usually a limited, and fixed number of possible values. However, these discrete values have a certain order, and the ordering is important to understanding the values. Ordinal variable types can be represented as strings, or integers. \n",
+    "\n",
+    "- **physical type**: `category`\n",
+    "- **standard tags**: `{'category'}`\n",
+    "- **parameters**:\n",
+    "    - `order` - the order of the ordinal values in the column from low to high\n",
+    "- **validation** - an order must be defined for an Ordinal column on a DataFrame or Series, and all elements of the order must be present.\n",
+    "- **koalas note**: Koalas does not support the `category` dtype, so for Koalas DataFrames and Series, the `string` dtype will be used.\n",
+    "\n",
+    "Some examples of data for which the Ordinal logical type would apply:\n",
+    "\n",
+    "- Educational Background (Elementary, High School, Undergraduate, Graduate)\n",
+    "- Satisfaction Rating (“Not Satisfied”, “Satisfied\", “Very Satisfied”)\n",
```

Comment:
Since we didn't use quotes for the other examples, maybe remove them here as well?

Reply:
will do!